lib/chef/provider/aws_internet_gateway.rb

```diff
@@ -0,0 +1,78 @@
+require 'chef/provider/aws_provider'
```

Comment:
@jkeiser core chef attaches an apache2 header to the top of all files - do we do the same here?